hassio/src/system/hassio-host-info.ts

```diff
@@ -150,6 +150,17 @@ class HassioHostInfo extends LitElement {
               : ""}
           </div>
           <div>
+            ${this.supervisor.host.disk_life_time
```

Comment:
If it is `0` we don't show it now, is that intended?

Reply:
the lowest value returned is 10

Reply:
that's an assumption :D I would check against undefined.

Reply:
Undefined is not used, but i get your point